R/eula.R

```diff
@@ -31,7 +31,7 @@ eula <- function() {
   while (!(resp %in% c("y", "yes", "n", "no"))) {
     prompt <- paste0(
       prompt_begin,
-      "Do you accept the End-User License Agreement\n(y/yes or n/no): ",
+      "Do you accept the End-User License Agreement\n(y/yes or n/no): "
```

Comment:
Is there linting/formatting we can add to catch this problem in future?

Reply:
Added a ticket here: https://github.com/10XGenomics/loupeR/issues/77